config/locales/en.yml

```diff
@@ -1133,9 +1133,13 @@ en:
       this_is_private_community: "You need to sign up before you can view the content."
     invitation_form:
       email: "Email address(es)"
+      email_with_limit: "Email addresses (max. %{invitation_limit})"
       message: "A personal message"
       send_invitation: "Send invitation"
       add_as_many_emails_as_you_want: "Add the email addresses of the people you are inviting to the field below. If you add multiple email addresses, separate them with comma."
```

Comment:
True. However, I kind of left this because I would prefer to just rename the key. (The texts are almost the same - and therefore it would have been ok to just rephrase the old translation.)